docker/kubernetes-tentacle/Dockerfile

```diff
@@ -27,17 +27,26 @@ WORKDIR /tmp
 
 # Install Tentacle
 COPY _artifacts/docker/tentacle_${BUILD_NUMBER}_${TARGETOS}-${TARGETARCH}${TARGETVARIANT}.deb /tmp/tentacle.deb
-RUN apt-get update
-RUN apt install ./tentacle.deb -y
-RUN apt-get clean
-RUN apt install curl -y
-RUN rm -rf /var/lib/apt/lists/*
+RUN apt-get update  \
+    && apt install ./tentacle.deb curl -y  \
+    && apt-get clean  \
+    && rm -rf /var/lib/apt/lists/*
 
 WORKDIR /
 
 # We know this won't reduce the image size at all. It's just to make the filesystem a little tidier.
 RUN rm -rf /tmp/*
 
+# Make the /.dotnet directory for certificate store purposes
+RUN mkdir /.dotnet
+
+# This is to run on Openshift with a random user
```

Comment:
Can we add a link to the documentation as a comment here plz?

Reply:
Done in [58fa6e3](https://github.com/OctopusDeploy/OctopusTentacle/pull/914/commits/58fa6e3a7db77130d8c285beac41475317b6dc71)!